tests/jsx_escape/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -25,7 +25,7 @@ raw_amp = <span>foo & bar</span>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 many_nbsp = <div>&nbsp; &nbsp; </div>;
 single_nbsp = <div>&nbsp;</div>;
-many_raw_nbsp = <div>&nbsp;&nbsp;&nbsp;</div>;
+many_raw_nbsp = <div> </div>;
```

Comment:
In this case, three spaces were formerly transformed as multiple nbsp; Now the spaces are conserved but only as one space. Somehow, that doesn't make sense to keep n spaces. As soon as spaces are mixed with anything else, they are not dropped by the parsers(of course spaces put in a row are going to be replaced by one).